beacon_node/lighthouse_network/src/types/globals.rs

```diff
@@ -33,6 +33,8 @@ pub struct NetworkGlobals<E: EthSpec> {
     /// The computed sampling subnets and columns is stored to avoid re-computing.
     pub sampling_subnets: HashSet<DataColumnSubnetId>,
     pub sampling_columns: HashSet<ColumnIndex>,
+    /// Constant custody group count (CGC) set at startup
+    custody_group_count: u64,
```

Comment:
You think it might be worth using one naming convention (like `custody_group_count`) everywhere instead of also using `custody_columns_count`?

Reply:
Also this isn't behind any `RwLock` or anything but it was my understanding that this could change as the node synced more columns / added more validators? Or is that just not implemented yet?

Reply:
not implemented yet